### ISSUE TITLE
systemd unit broken on ubuntu .deb packages

### ISSUE THREAD

**Issue by s-mittelberger-pexta** [emqx/emqx#7825]
### systemd unit broken on ubuntu .deb packages

The systemd unit fails to restart after an update from emqx 4.3.12 to 4.3.14 with the following error:
```
# systemctl restart emqx
Failed to restart emqx.service: Unit emqx.service is not loaded properly: Exec format error.
See system logs and 'systemctl status emqx.service' for details.
```

The status command returns:
```
# systemctl status emqx
[...]
Apr 28 16:59:29 vsmqtttest01 systemd[1]: /lib/systemd/system/emqx.service:14: Executable path is not absolute: bash /usr/bin/emqx start
```

The reason for this is the recently introduced bash wrapper which lack the absolute path for the binary.

The broken version contains:
```
[...]
ExecStart=bash /usr/bin/emqx start
[...]
ExecStop=bash /usr/bin/emqx stop
[...]
```

It works if you change it to:
```
[...]
ExecStart=/bin/bash /usr/bin/emqx start
[...]
ExecStop=/bin/bash /usr/bin/emqx stop
[...]
```

### Your environment

For EMQ X 4.3 or newer, please provide the log archive generated by running `node_dump` (The `node_dump` script can be found in `/<path-to-installation-of-emqx>/bin` if you installed emqx from source or zip ball).

- EMQ X version (e.g. `emqx_ctl status`): 4.3.14
- Hardware configuration (e.g. `lscpu`): not relevant
- OS (e.g. `cat /etc/os-release`): not relevant
- Kernel (e.g. `uname -a`): not relevant
- Erlang/OTP version (in case you build emqx from source code): not relevant
- Others: not relevant

**Comment by zmstone** [comment]
Thank you for the issue report @s-mittelberger-pexta 
We'll fix it in the next release.

**Comment by zmstone** [comment]
fix has been merged, will be a part of 4.3.15 and 4.4.4
